cypress/integration/tests/controls/businessservices/crud.test.ts

```diff
@@ -25,7 +22,7 @@ describe("A single Business service", () => {
         cy.wait("@postBusinessService");
 
         // Edit Business service's name
-        var updatedBusinessServiceName = data.getBusinessServiceName();
+        var updatedBusinessServiceName = data.getCompanyName();
         businessService.edit({ name: updatedBusinessServiceName });
```

Comment:
If we are not using this var `updatedBusinessServiceName` then it will be better to pass that directly.

Reply:
It will be used, for verifying the edit operation assert, which should have been done in original PR. Anyways, I will add here.

Reply:
Used the var for edit verification